### ISSUE TITLE
Cannot use ListField as mutation input

### ISSUE THREAD

**Issue by rapilabs** [graphql-python/graphene#27]
I'm porting a graphql server that was created with your graphql-relay js port.  I have a mutation that has a list field as an input and works fine, however when I try to create a list field with graphene I get the following error:   

```
AssertionError: NewCellInnerInput.newCells field type must be Input Type but got: [Cell].
```

Below is a snippet from my ported schema:

```python
class Cell(graphene.ObjectType):
    title = graphene.StringField()
    item = graphene.StringField()


class NewCell(relay.ClientIDMutation):
    class Input:
        parent = graphene.IDField(required=True)
        newCells = graphene.ListField(Cell)

    parent = graphene.Field(Tree)
```

**Comment by jhgg** [comment]
Hi @rapilabs,

Issue here is that you are trying to use an `ObjectType` as an `InputType`. They are not compatible. This is a limitation with graphql in general, and not an issue with `graphene`. 

What you would have to do is define a Cell that is an `InputObjectType `.

Perhaps:

```python
class CellInput(graphene.InputObjectType):
    title = graphene.StringField()
    item = graphene.StringField()
```

**Comment by rapilabs** [comment]
@jhgg, mate, props for the instant response on a weekend!

Ok yes I see now, it's working thanks!

Note I had to actually manually import `InputObjectType` from `graphene.core.types` as it's not part of the `graphene` namespace: https://github.com/graphql-python/graphene/blob/master/graphene/__init__.py#L15

**Comment by syrusakbary** [comment]
@rapilabs this will be fixed in the upcoming version 0.4.0 ;)
